apis/apps/v1alpha1/cluster_types.go

```diff
@@ -354,7 +357,7 @@ func (r PersistentVolumeClaimSpec) ToV1PersistentVolumeClaimSpec() corev1.Persis
 	return corev1.PersistentVolumeClaimSpec{
 		AccessModes:      r.AccessModes,
 		Resources:        r.Resources,
-		StorageClassName: r.StorageClassName,
+		StorageClassName: r.GetStorageClassName(viper.GetString(constant.CfgKeyDefaultStorageClass)),
```

Comment:
should check if PersistentVolumeClaimSpec contains specified storageClass

Reply:
`r.GetStorageClassName` checked it.

```go
// GetStorageClassName returns PersistentVolumeClaimSpec.StorageClassName if a value is assigned; otherwise,
// it returns preferSC argument.
func (r PersistentVolumeClaimSpec) GetStorageClassName(preferSC string) *string {
	if r.StorageClassName != nil && *r.StorageClassName != "" {
		return r.StorageClassName
	}
	return &preferSC
}